manifests/env.pp

```diff
@@ -6,21 +6,21 @@
 # parameter
 
 define poudriere::env (
-  $ensure           = 'present',
-  $makeopts         = [],
-  $makefile         = nil,
-  $version          = '10.0-RELEASE',
-  $arch             = 'amd64',
-  $jail             = $name,
-  $paralleljobs     = $facts['processors']['count'],
-  $pkgs             = [],
-  $pkg_file         = nil,
-  $pkg_makeopts     = {},
-  $pkg_optsdir      = nil,
-  $portstree        = 'default',
-  $cron_enable      = false,
-  $cron_always_mail = false,
-  $cron_interval    = { minute => 0, hour => 0, monthday => '*', month => '*', weekday => '*' },
+  Enum['present', 'absent'] $ensure           = 'present',
+  Array[String]             $makeopts         = [],
```

Comment:
I guess empty strings aren't allowed?
```suggestion
  Array[String[1]]          $makeopts         = [],
```

Reply:
Yes, a bunch of `String` should be `String[1]`